volmdlr/wires.py

```diff
@@ -491,6 +491,36 @@ def random_point_inside(self):
     #     else:
     #         raise NotImplementedError('Non convex contour not supported yet')
 
+    def cut_by_simple_contour(self, contour: 'Contour2D'):
+        for c in self.primitives:
```

Comment:
What is the purpose of this method? Why not calling it cut_by_linesgment?

Reply:
change in current commiut by cut_by_linesegments